langchain/src/util/prompt-layer.ts

```diff
@@ -0,0 +1,42 @@
+import fetch from 'cross-fetch';
+import { AsyncCaller } from "../util/async_caller.js";
+
+export const getPromptLayerRequestID = async (
```

Comment:
@SalehHindi is there a reason why this function is called `getPromptLayerRequestID`? Why is the name focusing on getting the request ID when the main functionality of this function is to make a promptlayer track-request.

Reply:
I tried making the function narrow in scope but I can change it so that it's something like `makePromptLayerTrackRequest` or `promptLayerTrackRequest`. I think now that you say that, that makes more sense.

Reply:
Agreed on this, can you rename?

It's also more conventional on our end to do `getPromptLayerRequestId` with the "D" in `id` lower-cased.

Reply:
Agreed on the name change. Changed the function name to `promptLayerTrackRequest` which is typically called like `let promptLayerRespBody = await promptLayerTrackRequest(...)`. Lmk if there is something better you suggest. 

Also thx for the tip about naming id like `varNameId` instead of `varNameID`.